5-infrastructure/modules/base-env/outputs.tf

```diff
@@ -42,24 +42,33 @@ output "kms_outputs" {
 *****************************************/
 
 output "bastion_hostname" {
-  description = "Host name of the bastion"
+  description = "Host name of the bastion."
   value       = module.bastion.hostname
 }
 
 output "bastion_ip_address" {
-  description = "Internal IP address of the bastion host"
+  description = "Internal IP address of the bastion host."
   value       = module.bastion.ip_address
 }
 
 output "bastion_service_account_email" {
-  description = "Email address of the SA created for the bastion host"
+  description = "Email address of the SA created for the bastion host."
   value       = module.bastion.service_account_email
 }
 
 /******************************************
   GKE Outputs
 *****************************************/
 output "gke_outputs" {
-  description = "Outputs for Cloud SQL instances"
+  description = "Outputs for Cloud SQL instances."
   value       = tomap({ for key, cluster in module.clusters : key => { "Cluster Name" = cluster.name, "Region" = cluster.region, "Master IPV4 Address CIDR" = cluster.master_ipv4_cidr_block } })
 }
+
+/******************************************
+  External IP Outputs
+*****************************************/
+
+output "external_ip_address" {
```

Comment:
I assume we will need to plumb this through for each env @rutalreja-deloitte 
We can do that in a separate PR